lib/server.js

```diff
@@ -1170,22 +1173,24 @@ Server.prototype._setupRequest = function _setupRequest(req, res) {
         req._timeUseEnd = req._timeUseEnd || processHrTime;
         req._timeRouteEnd = req._timeRouteEnd || processHrTime;
 
-        req.removeListener('close', onClose);
-        res.removeListener('error', onError);
-
-        // Do not trigger false
         self._finishReqResCycle(req, res);
     }
-    function onError(err) {
+
+    // Emitted when the response has been sent.
```

Comment:
Is this still true? Also, I think I know the answer, but is it possible for close to get fired but `_flushed` is false?